runeq/resources/user.py

```diff
@@ -0,0 +1,153 @@
+"""
+Fetch metadata about Rune platform users.
+
+"""
+
+from typing import Optional
+
+from .client import GraphClient, global_graph_client
+from .common import ItemBase
+from runeq import Config
+from runeq.resources import client
+
+
+def initialize(*args, **kwargs):
```

Comment:
This doesn't belong in here, IMO. Can you stick it in the existing `config.py`, and also import it in the top-level `__init__.py` (so that we can do `from runeq import initialize`)?

Reply:
Moving initialize to `config.py` causes a circular import since it requires `client.py`, and `client.py` needs `Config`. Opted to move `initialize` to `client.py` instead.